playgroundFaq.md

```diff
@@ -9,61 +9,121 @@ Have you already integrated CHAPI into your project and now you’re interested
-3. The CHAPI Playground \exchanges endpoint responds with a VP containing the specified credential
+The general /exchanges pattern is illustrated in the [Verifiable Presentation Request (VPR) spec](https://w3c-ccg.github.io/vp-request-spec/#example-example-interact-request), which shows how an issuer can ask for DID Authentication and then point the wallet at /exchanges endpoint to perform the Auth-Pickup workflow.
 
-Example data for each step can be found [here](https://gist.github.com/tolson4/6e44720a15957136fbe6ca33403c4bcb).
+In this example, the `interact` field describes the type and URL of the /exchanges endpoint.  The `type` is an `UnmediatedPresentationService2021`, which is outlined in the [VCAPI spec](https://w3c-ccg.github.io/vc-api/#exchange-examples).
 
-We’ve added an example to the [Verifiable Presentation Request (VPR) spec](https://w3c-ccg.github.io/vp-request-spec/#example-example-interact-request) to show how an issuer can ask for DID Authentication and then point the wallet at /exchanges endpoint to perform the Auth-Pickup workflow.
+The CHAPI Playground implementation follows the general pattern, via these spcific steps (with example data):
+
+**Step 1**: CHAPI Playground sends a DID Auth Request (VPR) over CHAPI to the wallet.  This includes an `interact` field that specifies an /exchanges endpoint on the CHAPI Playground.  
+```
+{
+    "query": {
+        "type": "DIDAuthentication"
+    },
+    "interact": {
+        "service": [
+            {
+                "type": "VerifiableCredentialApiExchangeService",
+                "serviceEndpoint": "https://playground.chapi.io/exchanges/eyJjcmVkZW50aWFsIjoiaHR0cHM6Ly9wbGF5Z3JvdW5kLmNoYXBpLmlvL2V4YW1wbGVzL2pmZjIvamZmMi5qc29uIiwiaXNzdWVyIjoiZGIvdmMifQ/esOGVHG8d44Q"
+            },
+            {
+                "type": "CredentialHandlerService"
+            }
+        ]
+    },
+    "challenge": "g2Wvqg7-cQGWYCCH55rUl",
+    "domain": "https://playground.chapi.io"
+}
+```
+
+**Step 2**: The Wallet POSTs a VP containing the DID Auth to the /exchanges endpoint.
+```
+{
+  "verifiablePresentation" :{
+    "@context": [
+      "https://www.w3.org/2018/credentials/v1"
+    ],
+    "type": [
+      "VerifiablePresentation"
+    ],
+    "holder": "did:example:z6MkjsQSCqdN4CGE6R9tKhETAEoPYdcpO5v4tK2USAhWptpr",
+    "proof": {
+      "type": "Ed25519Signature2020
+      "created": "2022-10-17T18:12:55Z",
+      "verificationMethod": "did:example:z6MkjsQSCqdN4CGE6R9tKhETAEoPYdcpO5v4tK2USAhWptpr#z6MkjsQSCqdN4CGE6R9tKhETAEoPYdcpO5v4tK2USAhWptpr",
+      "proofPurpose": "authentication",
+      "challenge": "TTkKgear2sBt4TYxr_zrL",
+      "domain": "https://playground.chapi.io",
+      "proofValue": "eyJhbGciOiJFZERTQSIsImI2NCI6ZmFsc2UsImNyaXQiOlsiYjY0Il19..gaG1RaxRy-xLkjVN-7ulh0dbUU1EX"
+    }
+  }
+}
+```
+
+**Step 3**: The CHAPI Playground /exchanges endpoint responds with a VP containing the specified credential
+```
```

Comment:
Using `jsonc` due to comment
````suggestion
```jsonc
````

Reply:
Formatting looks weird with our jekyll template - keeping as a generic code block.